source/common/access_log/BUILD

```diff
@@ -61,3 +34,30 @@ envoy_cc_library(
         "@envoy_api//envoy/config/filter/accesslog/v2:accesslog_cc",
     ],
 )
+
+envoy_cc_library(
```

Comment:
Looks like these 2 stanzas were moved for no reason. Revert? Or at least move them back above access_log_lib so the diffs are easier to see.